src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -607,6 +611,47 @@ const VisitorSpaceSearchPage: FC = () => {
 		);
 	};
 
+	const renderTempAccessLabel = () => {
+		return (
+			<div className="p-visitor-space__temp-access-container">
+				<Icon name={IconNamesLight.Clock} />
+				<span className="p-visitor-space__temp-access-label">
+					{tText(
+						'modules/visitor-space/components/visitor-space-search-page/visitor-space-search-page___tijdelijke-toegang'
+					)}
+					{dropdownOptions
+						// Ward: remove 'Publieke catalogus'
+						.filter((maintainer) => maintainer.id !== '')
```

Comment:
also filter out the visitor space of the user if they are a cp admin or kiosk user
![image](https://user-images.githubusercontent.com/1710840/226560017-36ecb91a-5efa-4e5e-b2b6-f0c9720d7433.png)